groupsforsections.py

```diff
@@ -218,6 +245,7 @@ def main():
 
     # instantiating the class
     groups_for_section_class = GroupsForSections(canvas_token, canvas_url)
+    # this hold the list of users that needs to be added to a group, group => users
```

Comment:
Does this mean: this holds a set of groups where each group holds the list of users to be added to the group?